spring-boot-tools/spring-boot-loader-tools/src/main/java/org/springframework/boot/loader/tools/RunProcess.java

```diff
@@ -58,6 +66,7 @@ public int run(boolean waitForProcess, String... args) throws IOException {
 	protected int run(boolean waitForProcess, Collection<String> args)
 			throws IOException {
 		ProcessBuilder builder = new ProcessBuilder(this.command);
+		builder.directory(workingDirectory);
```

Comment:
Now that I've looked at the javadoc of `ProcessBuillder#directory` I take it back: it is very explicit about the semantic of `null` there so I'd keep it like that. I'd add a javadoc on the constructor to reflect that information (but I can take care of that myself when I merge).

Feel free to update this PR, just push an additional commit on your branch (or preferably, rebase + push force)